default.nix

```diff
@@ -0,0 +1,7 @@
+{ pkgs ? import <nixpkgs> {} }:
+let
+  result = import /home/dmj/miso {};
```

Comment:
We'll need to get an equivalent hash at the revision. I recommend copying it in, letting it fail (it will tell you what hash it was expecting), and then copy the correct one in.